modules/geo-layers/src/mvt-layer/mvt-layer.js

```diff
@@ -70,13 +72,13 @@ export default class MVTLayer extends TileLayer {
     if (tileJSON) {
       data = tileJSON.tiles;
 
-      if (Number.isFinite(tileJSON.minzoom) && tileJSON.minzoom > minZoom) {
+      if (Number.isFinite(tileJSON.minzoom) && tileJSON.minzoom > this.props.minZoom) {
```

Comment:
This is still depending on a prop that could later change. This chunk pausing should be performed in getTilesetOptions

Reply:
Good catch, thanks, another approach implemented.